micro-ui/web/packages/modules/workbench/src/components/BulkUploadNew.js

```
@@ -0,0 +1,205 @@
+    ));
+  }, [fileData, fileUrl]);
+
+  return (
+    <React.Fragment>
+      {(!fileData || fileData?.length === 0) && (
+        <FileUploader
+          multiple={multiple}
+          handleChange={handleChange}
+          name="file"
+          types={fileTypes}
+          children={dragDropJSX}
```

Biome (2.1.2)

[error] Avoid passing children using a prop

The canonical way to pass children in React is to use JSX elements

(lint/correctness/noChildrenProp)